charts/digger-backend/values.yaml

```diff
@@ -6,7 +6,7 @@ digger:
   # tag: digger backend image tag
   image:
     repository: registry.digger.dev/diggerhq/digger_backend
-    tag: "v0.4.9"
+    tag: "latest"
```

Comment:
The problem with using "latest" is that it no longer appears as a change in the values when helm reconciles, so it ends up actually meaning "whatever was current when the chart was installed".  Which is very much not what is intended.